src/crystal/system/win32/process.cr

```diff
@@ -83,7 +83,14 @@ struct Crystal::System::Process
   end
 
   def self.times
-    raise NotImplementedError.new("Process.times")
+    if LibC.GetProcessTimes(LibC.GetCurrentProcess, out create, out exit, out kernel, out user) == 0
```

Comment:
Since `LibC.GetProcessTimes` returns `Bool` this will never enter into this branch.

Reply:
`alias BOOL = Int32`